mathics/builtin/comparison.py

```diff
@@ -630,11 +636,15 @@ class Equal(_EqualityOperator, _SympyComparison):
     >> Pi ^ E == E ^ Pi
      = False
 
-    ## TODO needs better precision tracking
-    ## #> 2^^1.000000000000000000000000000000000000000000000000000000000000 ==  2^^1.000000000000000000000000000000000000000000000000000001111111
-    ##  = True
-    ## #> 2^^1.000000000000000000000000000000000000000000000000000000000000 ==  2^^1.000000000000000000000000000000000000000000000000000010000000
-    ##  = False
+    Compare an exact expression against an approximate real number:
+    >> Pi == 3.1415``4
+     = True
+
+    #> 2^^1.000000000000000000000000000000000000000000000000000000000000 ==  2^^1.000000000000000000000000000000000000000000000000000000000001
```

Comment:
This would be better moved to a pytest. (Yes, I understand this is old cleanup).